ui/package.json

```diff
@@ -142,16 +142,13 @@
     "ember-cli-terser": "^4.0.2",
     "ember-cli-typescript": "^5.2.1",
     "ember-composable-helpers": "5.0.0",
-    "ember-concurrency": "2.3.4",
-    "ember-copy": "2.0.1",
-    "ember-d3": "^0.5.1",
+    "ember-concurrency": "^3.1.1",
     "ember-data": "~4.12.4",
     "ember-engines": "0.8.23",
     "ember-exam": "^9.0.0",
     "ember-fetch": "^8.1.2",
     "ember-inflector": "4.0.2",
     "ember-load-initializers": "^2.1.2",
-    "ember-maybe-in-element": "^2.0.3",
```

Comment:
from @meirish in [this comment](https://github.com/hashicorp/vault/pull/24646/files#r1437790395):
> This probably took the longest to figure out. I kept getting the error `unable to use this non-serializable babel config in this node process` but the stack trace wasn't very helpful.
> 
> This issue: [embroider-build/embroider#1525](https://github.com/embroider-build/embroider/issues/1525) is the same as I was seeing, and adding the `throwUnlessParallelizable: true,` as described in that thread didn't print the name of the addon, it just listed "unknown". So after some trial and error and thinking about what addons likely have babel transforms, I tried removing this, and it fixed the build 🎉.
> 
> `ember-maybe-in-element` won't be necessary anymore as recent ember has the `~in-element` helper included. So this is safe to remove.